README.md

```diff
@@ -43,6 +43,16 @@ bin/rails generate solidus_stripe:install
 
 ## Development
 
+Retrieve your API Key and Publishable Key from your [Stripe testing dashboard](https://stripe.com/docs/testing).
+
+Set `SOLIDUS_STRIPE_API_KEY` and `SOLIDUS_STRIPE_PUBLISHABLE_KEY` environment variables (e.g. via `direnv`), this
+will trigger the default initializer to create a static preference for SolidusStripe.
+
+Run `bin/dev` to start both the sandbox rail server and the file watcher that will update the sandbox whenever
```

Comment:
What if `bin/dev` also creates the payment method if needed? I think it would be a great UX improvement but don't know if that's easy to do. Thoughts?

Reply:
Added the static pref that for some reason was left out of the PR and bin/sandbox will auto-create the payment method connecting to the static pref if the same env var is present. 👍